packaging/irods_client_rest_cpp_reverse_proxy.conf.template.in

```diff
@@ -82,4 +82,12 @@ server {
 
         proxy_pass http://localhost:8086;
     }
+
+    location /irods-rest/@IRODS_CLIENT_VERSION@/logicalpath {
+        if ($request_method = 'OPTIONS') {
+            return 204;
+        }
+
+        proxy_pass http://localhost:8091;
```

Comment:
Is port 8090 already used?

Reply:
I think this was the result of some tinkering but no; will put on that more reasonable port number.